3/ollie-charles-captions.srt

```diff
@@ -4773,11 +4777,11 @@ To me, the biggest source of expense
 
 1095
 01:10:15,250 --> 01:10:16,902
-feels like this neighbors function.
+feels like this neighbours function.
```

Comment:
I was about to say "No, we're preferring American spellings to UK spellings", but then remembered that this is the name of a function Ollie wrote, so we'll copy his preferred spelling, which is British.